README.md

```diff
@@ -119,37 +119,46 @@ The macOS installation application is [code signed](https://developer.apple.com/
 To verify the code signature and integrity of macOS application bundles:
 
 ```console
-$ pkgutil --check-signature /Applications/Install\ macOS\ Catalina.app
-Package "Install macOS Catalina":
-   Status: signed by a certificate trusted by Mac OS X
+$ pkgutil --check-signature /Applications/Install\ macOS\ Big\ Sur.app
+Package "Install macOS Big Sur":
+   Status: signed by a certificate trusted by macOS
    Certificate Chain:
     1. Software Signing
-       SHA1 fingerprint: 01 3E 27 87 74 8A 74 10 3D 62 D2 CD BF 77 A1 34 55 17 C4 82
-       -----------------------------------------------------------------------------
+       Expires: 2026-10-24 17:39:41 +0000
+       SHA256 Fingerprint:
+           D8 4D B9 6A F8 C2 E6 0A C4 C8 51 A2 1E C4 60 F6 F8 4E 02 35 BE B1
+           7D 24 A7 87 12 B9 B0 21 ED 57
+       ------------------------------------------------------------------------
     2. Apple Code Signing Certification Authority
-       SHA1 fingerprint: 1D 01 00 78 A6 1F 4F A4 69 4A FF 4D B1 AC 26 6C E1 B4 59 46
-       -----------------------------------------------------------------------------
+       Expires: 2026-10-24 17:39:41 +0000
+       SHA256 Fingerprint:
+           5B DA B1 28 8F C1 68 92 FE F5 0C 65 8D B5 4F 1E 2E 19 CF 8F 71 CC
+           55 F7 7D E2 B9 5E 05 1E 25 62
+       ------------------------------------------------------------------------
     3. Apple Root CA
-       SHA1 fingerprint: 61 1E 5B 66 2C 59 3A 08 FF 58 D1 4A E2 24 52 D1 98 DF 6C 60
+       Expires: 2035-02-09 21:40:36 +0000
+       SHA256 Fingerprint:
+           B0 B1 73 0E CB C7 FF 45 05 14 2C
 ```
 
 Use the `codesign` command to examine an application's code signature:
 
 ```console
-$ codesign -dvv /Applications/Install\ macOS\ Catalina.app
-Executable=/Applications/Install macOS Catalina.app/Contents/MacOS/InstallAssistant_springboard
-Identifier=com.apple.InstallAssistant.Catalina
-Format=app bundle with Mach-O thin (x86_64)
-CodeDirectory v=20100 size=276 flags=0x2000(library-validation) hashes=3+3 location=embedded
-Platform identifier=9
-Signature size=4628
+$ codesign -dvv /Applications/Install\ macOS\ Big\ Sur.app
+Executable=/Applications/Install macOS Big Sur.app/Contents/MacOS/InstallAssistant_springboard
+Identifier=com.apple.InstallAssistant.macOSBigSur
+Format=app bundle with Mach-O universal (x86_64 arm64)
+CodeDirectory v=20400 size=511 flags=0x2000(library-validation) hashes=9+3 location=embedded
+Platform identifier=12
+Signature size=4523
 Authority=Software Signing
 Authority=Apple Code Signing Certification Authority
 Authority=Apple Root CA
-Info.plist entries=33
+Signed Time=10 Sep 2021 at 00:56:58
+Info.plist entries=32
 TeamIdentifier=not set
-Sealed Resources version=2 rules=13 files=234
-Internal requirements count=1 size=84
+Sealed Resources version=2 rules=2 files=0
+Internal requirements count=1 size=88
```

Comment:
Likewise fixed upstream:
https://github.com/drduh/macOS-Security-and-Privacy-Guide/commit/c64568e510774ca7aa1dddc21fd1d19b19c5e0dc#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R173-R174